src/providers/export/tensorFlowRecords.ts

```diff
@@ -183,10 +183,12 @@ export class TFRecordsJsonExportProvider extends ExportProvider<ITFRecordsJsonEx
 
                                         imageInfo.text.push(tag.name);
                                         imageInfo.label.push(index);
-                                        imageInfo.xmin.push(region.boundingBox.left);
-                                        imageInfo.ymin.push(region.boundingBox.top);
-                                        imageInfo.xmax.push(region.boundingBox.left + region.boundingBox.width);
-                                        imageInfo.ymax.push(region.boundingBox.top + region.boundingBox.height);
+                                        imageInfo.xmin.push(region.boundingBox.left / imageInfo.width);
```

Comment:
Why dividing by width/height here?

Reply:
I just figure it out TFRecords format is expecting relative points expressed in 0.0<=1.0 range respect to the image width and height

Reply:
OK - makes sense then